app/tests/spec/views/sign_in_unblock.js

```diff
@@ -217,6 +217,26 @@ define(function (require, exports, module) {
         });
       });
 
+      describe('incorrect unblock_code', () => {
+        const incorrectUnblockCodeError =
+          AuthErrors.toError('INCORRECT_UNBLOCK_CODE');
+
+        beforeEach(() => {
+          sinon.stub(view, 'signIn').callsFake(() => Promise.reject(incorrectUnblockCodeError));
+
+          sinon.stub(view, 'navigate').callsFake(() => Promise.resolve());
+
+          view.$('#unblock_code').val(UNBLOCK_CODE);
+
+          return view.submit();
+        });
+
+        it('displays a tooltip, does not call submit', () => {
+          assert.isTrue(view.incorrectUnblockCodeError.called);
```

Comment:
Instead of this, you should assert that the `showValidationError` was called with the correct jquery element and correct error. You can use sinon helper `calledWithExactly`.